### ISSUE TITLE
Creating a config file leads to the generation of an output directory

### ISSUE THREAD

**Issue by Jean-Du** [AFM-SPM/TopoStats#522]
**NB** TopoStats `< 2.0.0` is no longer actively maintained. If you have encountered an issue using this deprecated
version please try TopoStats `>= 2.0.0`, there are instructions on
[installation](https://afm-spm.github.io/TopoStats/installation.html),
[usage](https://afm-spm.github.io/TopoStats/usage.html) and
[configuration](https://afm-spm.github.io/TopoStats/configuration.html).

By default TopoStats will process images in parallel to speed up processing. In order to identify the problematic image
you should re-run your analyses using the `--cores 1` option so that images are processed sequentially. This makes it
possible to identify which image causes the problem.

## Checklist

Please try and tick off each of these items when filing the bug report. There are further instructions on each below.

* [ ] Re-run analysis with `run_topostats --core 1`.
* [ ] Describe the bug.
* [ ] Include the configuration file.
* [ ] Copy of the output.
* [ ] The exact command that failed. This is what you typed at the command line, including any options.
* [ ] TopoStats version, this is reported by `run_topostats --version`
* [ ] Operating System and Python Version

## Describe the bug
When the command `run_topostats --create-config-file config.yaml` is used, an empty directory named `output` is generated in addition to the config file. This doesn't happen if a directory named `output` already existed. 

## Copy of the output

Please copy and paste the output that is shown below within the `\`` (triple-backticks).

```
<-- PASTE OUTPUT HERE -->
```

## Include the configuration file.

If no configuration file was specified with the `-c`/`--config-file` option the defaults were used, please use the
`run_topostats --create-config-file crash.yaml` to save these to the `crash.yaml` file and copy the contents below.

``` yaml
<-- PASTE CONTENT OF crash.yaml HERE -->
```

## To Reproduce

If it is possible to share the file (e.g. via cloud services) that caused the error that would greatly assist in reproducing and investigating the problem. In addition the _exact_ command used that failed should be pasted below.

```
<-- PASTE FAILING COMMAND HERE -->
```

## Output

Any output files that have been produced can be attached to this bug report (by default they will be under the `output` directory unless you have customised the configuration).

## TopoStats version

Please report the version of TopoStats you are using. There are several ways of doing this, either with `pip` or
`run_topostats`. Please copy and paste all output from either of the following commands.

- `pip show topostats`
- `run_topostats --version`

[20ad187](https://github.com/AFM-SPM/TopoStats/commit/20ad18729299bbd109f2b1b950be2f386f8bda48)

### Operating System

Please let us know what operating system you are using, if you have used more than one then tick all boxes.

* [x] Windows
* [ ] MacOS Intel (pre-2021)
* [ ] MacOS M1/M2 (post-2021)
* [ ] GNU/Linux (please include distribution)

### Python Version

Please let us know the version of Python you are using, paste the results of `python --version`

```
3.9.12
```

### Optional : Python Packages

If you are able to provide a list of your installed packages that may be useful. The best way to get this is to copy and paste the results of typing `pip freeze`.

```
<-- PASTE PYTHON PACKAGE INFORMATION -->
```

## Additional context

Low priority as it doesn't affect the functioning of topostats. It might even be useful to keep this bug instead of fixing it, so that issue #201 can be prevented.

**Comment by ns-rse** [comment]
This is simple to fix.


Line 164 which creates the output directory...

```python
    config["output_dir"].mkdir(parents=True, exist_ok=True)
```

Should be moved to after the current lines 166-180 which create the sample configuration file.
